### DISCUSSION TITLE
How can I add color to my app name inside the notification

### DISCUSSION THREAD

**Discussion by Manouli** [invertase/react-native-firebase#4924] [Q&A]
Hello everyone. Any ideas how I can set color for my app name on android push notification?
Example below

![image](https://user-images.githubusercontent.com/6332957/108214277-e4ba2600-7127-11eb-88da-4bfe6f4c002a.png)

I m currently setting the icon color successfully using this:

`<meta-data
            android:name="com.google.firebase.messaging.default_notification_color"
            android:resource="@color/colorAccent"
            tools:replace="android:resource" />`

But it only affects the icon color. Any help will be greatly appreciated.

**Comment by anujmpec** [comment]
any update ?

**Reply by mikehardy** [reply]
@anujmpec you have posted a request for updates on a discussion closed 1.5 years ago. That is very confusing behavior. In addition, open source means if you see no updates, there are no updates, and asking for them just generates notifications that have no value for people, it is not encouraged https://hackernoon.com/i-thought-i-understood-open-source-i-was-wrong-cf54999c097b
